tools/cldr-code/src/main/java/org/unicode/cldr/util/NameType.java

```diff
@@ -0,0 +1,275 @@
+package org.unicode.cldr.util;
+
+import java.util.Locale;
+
+public enum NameType {
+    NONE,
+    LANGUAGE,
+    SCRIPT,
+    TERRITORY,
+    VARIANT,
+    CURRENCY,
+    CURRENCY_SYMBOL,
+    TZ_EXEMPLAR,
+    TZ_GENERIC_LONG,
+    TZ_GENERIC_SHORT,
+    TZ_STANDARD_LONG,
+    TZ_STANDARD_SHORT,
+    TZ_DAYLIGHT_LONG,
+    TZ_DAYLIGHT_SHORT,
+    KEY,
+    KEY_TYPE,
+    SUBDIVISION;
```

Comment:
These are the replacements for CLDRFile.NO_NAME, CLDRFile.LANGUAGE_NAME, etc. (which were integers, NOT enum)